app/modules/code_provider/code_provider_service.py

```
@@ -156,17 +156,13 @@ def __init__(self, sql_db=None):
-            repo_manager_enabled = (
-                os.getenv("REPO_MANAGER_ENABLED", "false").lower() == "true"
-            )
-            if repo_manager_enabled:
-                from app.modules.repo_manager import RepoManager
+            from app.modules.repo_manager import RepoManager
 
-                self.repo_manager = RepoManager()
-                logger.info("ProviderWrapper: RepoManager initialized")
+            self.repo_manager = RepoManager()
+            logger.info("ProviderWrapper: RepoManager initialized")
         except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)